catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java

```diff
@@ -94,6 +94,15 @@ public void testVariableBricks() {
 				dataContainer.getUserVariable("p2", sprite).getValue());
 	}
 
+	public void testCreatingNewVariableCancelling() {
+		solo.clickOnText("p2");
```

Comment:
When I use "solo.pressSpinnerItem(spinnerIndex, -position);" to click on "New...", the "NewDataDialog" does not appear. That's why I used "clickOnText".
Any suggestions?